tests/runtime/model_management/test_model_manager.py

```diff
@@ -1054,3 +1056,126 @@ def test_lazy_load_handles_temporary_errors():
                 assert manager._lazy_sync_timer is None
                 model = manager.retrieve_model(model_name)
                 assert model
+
+
+def test_lazy_load_true_local_models_dir_valid():
+    """When lazy_load_local_models is True and local_models_dir exists.
+    Check that the local_models_dir is pointing to the correct location
+    """
+
+    with TemporaryDirectory() as cache_dir:
+
+        with non_singleton_model_managers(
+            1,
+            {
+                "runtime": {
+                    "library": "sample_lib",
+                    "local_models_dir": cache_dir,
+                    "lazy_load_local_models": True,
+                    "grpc": {"enabled": True},
+                    "http": {"enabled": True},
+                    "training": {"save_with_id": False, "output_dir": cache_dir},
+                    "service_generation": {
+                        "package": "caikit_sample_lib"
+                    },  # This is done to avoid name collision with Caikit itself
+                },
```

Comment:
same for all tests

Reply:
I also feel the tests can use the signature defined within `test_model_manager_loads_local_models_on_init` instead of the `non_singleton_model_managers`.